dashboard/Dockerfile

```diff
@@ -3,9 +3,11 @@ FROM python:latest
 WORKDIR /app
 
 COPY requirements.txt .
-RUN pip install -r requirements.txt
+RUN pip3 install -r requirements.txt
+
+EXPOSE 8200
 
 COPY streamlit_app.py .
 COPY .streamlit /.streamlit
 
-CMD streamlit run streamlit_app.py
\ No newline at end of file
+CMD streamlit run streamlit_app.py --server.port 8200
```

Comment:
Why port 8200?

Reply:
Port 8501 was causing us some issues locally, possibly due to it being used so frequently, so I tweaked the port to run tests locally and just kept it as 8200. No difference in functionality.